Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -562,12 +561,12 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
         {
             if h == nil
             {
-                delegate!.chartValueNothingSelected?(self)
+                delegate?.chartValueNothingSelected?(self)
```

Comment:
Why leave `delegate != nil` in the if statement if you're going to use optional chaining?
Maybe 
```if callDelegate, let delegate = delegate { ... } ``` instead?

Reply:
👍